specification/nginx/resource-manager/readme.md

```diff
@@ -28,7 +28,22 @@ These are the global settings for the Nginx API.
 title: NginxManagementClient
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2024-11-01-preview
+tag: package-2025-03-01-preview
+```
+
+### Tag: package-2025-03-01-preview
+
+These settings apply only when `--tag=package-2025-03-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2025-03-01-preview'
+input-file:
+  - NGINX.NGINXPLUS/preview/2025-03-01-preview/swagger.json
+suppressions:
+  - code: GetCollectionResponseSchema
+    from: swagger.json
+    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Nginx.NginxPlus/nginxDeployments/{deploymentName}/wafPolicies"]
+    reason: This is by design to avoid high bandwidth consumption
```

Comment:
I don't understand this. How is it going to avoid high bandwidth consumption?

Reply:
General saving bandwidth is not a good idea not to follow RPC rules, further discussion may be needed.

Reply:
@TimLovellSmith It is by design that the WafPolicy_List operation omits the content property from individual resources. This content field has large binary data. So if a customer has like 10 of these resources, we could experience packet drops if we has the content fields in the response. So it was decided that the customer would have to use the GET API to get the binary data of each resource. 

We had this discussion too in the private repo PR - https://github.com/Azure/azure-rest-api-specs-pr/pull/21186#issuecomment-2701549262